### PR TITLE
Update 01-api-definition.md

### PR DESCRIPTION
**Pull request by tkurian123**
Adding Inventory Settings to GET RS

### CODE REVIEW

File: apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/01-api-definition.md [hunk 1]
```diff
@@ -797,6 +797,15 @@ GET /properties/v1/mycompany/1234
         "value": null,
       }
     ],
+      "inventorySettings": {
+            "rateAcquisitionType": "SELL_LAR",
+            "distributionModels": [
+                "HOTEL_COLLECT",
+                "EXPEDIA_COLLECT"
+            ],
+            "pricingModel": "OCCUPANCY_BASED_CHILDREN_ALWAYS_EXTRA",
```
Comment (a-jomills) [inline apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/01-api-definition.md]: Do we want to use this value in the the docs? I think most all the providers use "PER_DAY".

Comment (tkurian123) [reply apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/01-api-definition.md]: Good call out..I ll make the change and resubmit..

Comment (a-jomills) [reply apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/01-api-definition.md]: @tkurian123 Any update on this?

**Comment by tkurian123** [comment]
@a-jomills Have made the update here.

File: apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/01-api-definition.md [hunk 2]
```diff
@@ -797,6 +797,15 @@ GET /properties/v1/mycompany/1234
         "value": null,
       }
     ],
+      "inventorySettings": {
```
Comment (jhitzke) [inline apis/02-Product Management/03-Property-API-(Contact Expedia Before Adopting)/01-api-definition.md]: Indenting is a out. :)

**Pull request merged**